manifests/e/EclipseFoundation/EclipseIDEforJavaDevelopers/2024-12/EclipseFoundation.EclipseIDEforJavaDevelopers.installer.yaml

```
@@ -11,7 +11,7 @@ NestedInstallerFiles:
   - RelativeFilePath: eclipse\eclipse.exe
 Installers:
 - Architecture: x64
-  InstallerUrl: https://0ms.dev/mirrors/eclipse//technology/epp/downloads/release/2024-12/R/eclipse-java-2024-12-R-win32-x86_64.zip
+  InstallerUrl: https://www.eclipse.org/downloads/download.php?file=/technology/epp/downloads/release/2024-12/R/eclipse-java-2024-12-R-win32-x86_64.zip
   InstallerSha256: C61061441300285D4A9F62301148A07E8AADA0946D6A08ECCDEB1A9FEC6D7A52

```

```suggestion
  InstallerSha256: 83EE3964C29A54CF4987A6594872E94C8CF7ECDA614504F293E2B21E4D0B2AA2
```

(Automated response - build 990.)